openapi/openapi-client/src/main/openapi3/master_rest_v1.yaml

```diff
@@ -1019,6 +1044,21 @@ components:
         - fetchPort
         - replicatePort
 
+
+    WorkerInterruptionNotice:
+      type: object
+      properties:
+        host:
+          type: string
+          description: The host of the worker.
```

Comment:
Better to use the worker's unique ID. Sometimes there are multiple workers on the same host.